sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -649,6 +649,14 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val SHOW_STATS_IN_EXPLAIN =
```

Comment:
If the `sizeInBytes` affects the plan decision, I think it makes sense to let users see it. 

When the plan is not expected and the number is super large, they might turn on CBO or trigger the command to re-analyze the tables. Hiding it looks not right to me, even if the number is ugly. : )